Xplat/src/main/java/vazkii/botania/common/block/flower/functional/LooniumBlockEntity.java

```diff
@@ -392,9 +392,10 @@ public void writeToPacketNBT(CompoundTag cmp) {
 
 	public static void dropLooniumItems(LivingEntity living, Consumer<ItemStack> consumer) {
 		var comp = XplatAbstractions.INSTANCE.looniumComponent(living);
-		if (comp != null && !comp.getDrop().isEmpty()) {
-			consumer.accept(comp.getDrop());
-		}
+		if (comp != null)
+			if (!comp.getDrop().isEmpty() || comp.isDropNothing()) {
```

Comment:
This `if` only fails if the component is set to drop an empty stack, without dropNothing being set, right? Does that ever happen? To me, it looks like the case when the set drop is empty works like if dropNothing is set anyway, so maybe we don't need dropNothing at all and can just set the component with an empty stack?